.github/workflows/integration.yml

```diff
@@ -0,0 +1,17 @@
+name: Test
+
+on:
+  [pull_request]
+
+jobs:
+  Unit-Tests:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Can we add names to the steps for more clarity about what each of them is doing?
Ref : Like [this](https://github.com/academind/github-actions-course-resources/blob/main/Code/02%20Basics/05%20Practice%20Project%20(Finished)/.github/workflows/deployment1.yaml) or [this](https://github.com/academind/github-actions-course-resources/blob/main/Code/02%20Basics/05%20Practice%20Project%20(Finished)/.github/workflows/deployment2.yaml)